src/scss/components/_form-control.scss

```diff
@@ -16,26 +16,33 @@
     display: none;
   }
 
-  &[class*="error"] {
+  .error {
```

Comment:
This results in the CSS rule `.openforms-form-control .error` rather than `.openforms-form-control.error` - is that intentional?